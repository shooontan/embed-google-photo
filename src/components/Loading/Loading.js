// @flow
import * as React from 'react';

const Loading = () => <p>loading...</p>;

type Props = {
  loading: boolean,
};

const isLoadingCheck = ({ loading }: Props) => (loading ? <Loading /> : null);

export default isLoadingCheck;
