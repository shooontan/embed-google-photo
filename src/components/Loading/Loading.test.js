// @flow
import * as React from 'react';
import TestRenderer from 'react-test-renderer';
import Loading from './Loading';

test('Form Component', () => {
  let testRenderer = TestRenderer.create(<Loading loading />);
  let testTree = testRenderer.toJSON();
  expect(testTree).toMatchSnapshot();

  testRenderer = TestRenderer.create(<Loading loading={false} />);
  testTree = testRenderer.toJSON();
  expect(testTree).toMatchSnapshot();
});
