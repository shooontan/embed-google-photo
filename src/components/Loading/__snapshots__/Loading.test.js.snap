// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Component 1`] = `
<p>
  loading...
</p>
`;

exports[`Form Component 2`] = `null`;
